### PR TITLE
Attempt to get working under PHP 8.0 and 8.1

### PR DESCRIPTION
**Pull request by nigelgbanks**
Upgrade symphony dependencies to ^4.4 to match up with Drupal 9.

Gonna test with all the other services at once, will post my results.

PHP is stopping security updates for 7.4 in November.

https://www.php.net/supported-versions.php

### CODE REVIEW

**Comment by whikloj** [comment]
Hey @nigelgbanks you are welcome to work on this, but I wanted to note that #133 is using Symfony 4.4 for all the microservices. It doesn't currently test PHP 8 but it might be a lower bar.

**Comment by nigelgbanks** [comment]
Hi @whikloj, thanks for pointing that out, at the moment I'm testing infrastructure changes that require a move to PHP 8.1 I'll probably stick to working against 2.x until 3.x is officially released.

**Comment by whikloj** [comment]
I understand, I'm just not super confident on when that will happen.

**Comment by nigelgbanks** [comment]
Just a heads up for 8.1 tests are failing due to guzzle https://github.com/guzzle/psr7/blob/8f0a09ce19adcf28f25ff9d90e7329488501b363/src/StreamWrapper.php#L47 is passing a `NULL` where it should be `FALSE` as PHP 8.1 has deprecated this. I'll need to get that sorted first. I tested with patching locally but I don't think we want to maintain a patch, better to fix it at the source.

**Comment by nigelgbanks** [comment]
So looking in the previous comment it would force me to update chullo which would affect crayfish-commons which would affect the islandora module, this would force users onto Drupal 9.4 and later. So I'll just patch it.

**Comment by nigelgbanks** [comment]
Can't easily patch it I'll just change the setting to ignore deprecations in the unit tests.

**Comment by nigelgbanks** [comment]
@whikloj it's finally ready for review :upside_down_face:

File: .scripts/tester [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-
+export SYMFONY_DEPRECATIONS_HELPER=disabled
 FAIL=0
 for D in */; do
   cd $D 
   echo "Operating on $D"
   composer test
   if [ $? -ne 0 ]; then 
     FAIL=1
+    exit 1
```
Comment (whikloj) [inline .scripts/tester]: The original point of the `FAIL` variable was to allow all the microservices to run their tests before we reported any failures. If you want to fail on the first error instead then you could remove it entirely.

Comment (nigelgbanks) [reply .scripts/tester]: My bad, that's my debug code slipping through, I just wanted a faster test loop and forgot to take it out. I'll remove it.

**Comment by whikloj** [comment]
I'm sorry @nigelgbanks, I'm trying to build a vagrant box with 8.0 to test this right now.

**Review by whikloj — APPROVED** [review]
Tested with PHP 8.0 and 8.1 👍

**Pull request merged**